tools/genhtml.pl

```diff
@@ -179,33 +206,35 @@ ()
                         }
                         if ( $flag2 == 1 ) {
 			    $row_line = $row_line . "$line \n";
-			    if ($line =~ /\ TFAIL\ / ) {
+
```

Comment:
I would rather be more robust and check for two variants (newlib vs oldlib) rather than loosening the expression.

So what about we do something as `if ($line =~ /\ TFAIL\ / or $line =~ /\ FAIL:/)` which is more robust and works for both oldlib and newlib tests.